IndexTopTen.md

```diff
@@ -69,7 +69,7 @@ This cheat sheet will help users of the [OWASP Top Ten](https://owasp.org/www-pr
 ## [A09:2021 – Security Logging and Monitoring Failures](https://owasp.org/Top10/A09_2021-Security_Logging_and_Monitoring_Failures/)
 
 * [Logging Cheat Sheet](cheatsheets/Logging_Cheat_Sheet.md)
-* [Application Logging Vocabulary Cheat Sheet](cheatsheets/Application_Logging_Vocabulary_Cheat_Sheet.md)
+* [Application Logging Vocabulary Cheat Sheet](cheatsheets/Logging_Vocabulary_Cheat_Sheet.md)
```

Comment:
Note: This file was renamed in a previous pull request, but it appears that this link wasn't updated. I updated the link the the existing file. Another possible option is to revert the previous  file name change.